server/util.go

```diff
@@ -116,25 +125,25 @@ func bindFlags(basename string, cmd *cobra.Command, v *viper.Viper) (err error)
 // is used to read and parse the application configuration. Command handlers can
 // fetch the server Context to get the CometBFT configuration or to get access
 // to Viper.
-func InterceptConfigsPreRunHandler(cmd *cobra.Command, customAppConfigTemplate string, customAppConfig interface{}, cmtConfig *cmtcfg.Config) error {
+func CreateServerContextFromConfig(cmd *cobra.Command, customAppConfigTemplate string, customAppConfig interface{}, cmtConfig *cmtcfg.Config) (*Context, error) {
```

Comment:
Do you think this conveys enough what the function does? (which is more than creating a server context, as it calls `interceptConfigs`).

I feel like maybe `InterceptConfigsPreRunHandlerNoLoggerNoSaveContext`, while hideous, will convey more what this does.

Reply:
Actually, I even think, this could be named `InterceptConfigsPreRunHandler` and have the other one be renamed to something more clear. However doing that is API breaking, so not ideal.

Reply:
Updated to `InterceptConfigsAndCreateContext`.